lib/spark_api/models/listing.rb

```diff
@@ -235,6 +235,10 @@ def respond_to?(method_symbol, include_all=false)
         end
       end
 
+      def batch_photo_delete(photoIds, args={})
```

Comment:
This sounds like a `!` method to me, especially because this class already has a `reorder_photo!` method that seems less... banging than this.

I'm not sure if you are familiar with the Ruby convention around bang methods, but [the language docs say](https://docs.ruby-lang.org/en/trunk/syntax/methods_rdoc.html):

> The bang methods (! at the end of method name) are called and executed just like any other method. However, by convention, a method with an exclamation point or bang is considered dangerous.

There are various interpretations of what "dangerous" means, but I'm pretty sure this qualifies by just about any measure.